src/event/util/log/remove.rs

```diff
@@ -27,10 +27,10 @@ fn remove_array(
 ) -> Option<(Value, bool)> {
     match path.next()? {
         PathComponent::Index(index) => match path.peek() {
-            None => array_remove(array, index).map(|v| (v, false)),
+            None => array_remove(array, index).map(|v| (v, array.is_empty())),
             Some(_) => array
                 .get_mut(index)
-                .and_then(|value| Some((remove_rec(value, path, prune)?.0, false))),
+                .and_then(|value| remove_rec(value, path, prune)),
```

Comment:
This makes it so that arrays can also be pruned, similar to maps.

Reply:
Ah I did this in the lookup work too. :) Good choice.